app/src/main/java/com/example/appempty/SecondFragment.kt

```diff
@@ -27,16 +27,17 @@ class SecondFragment : Fragment() {
 
 
        userViewModel.usuario.observe(this, androidx.lifecycle.Observer { result ->
-           fullname.text = result.name?.first ?: "No First Name"
-           username.text = result.login?.username ?: "No User Name"
-           state.text = result.location?.state ?: "No State"
-           email.text = result.email
+          fullname.text = result[0].name?.first ?: "No First Name"
```

Comment:
create a val for result[0], you should not be making this is a bad practice, if someone changes to 1 you will need a magnifier to find out where is the wrong index. instead of result[0] you can use result.first()

Reply:
Done